### PR TITLE
workaround and fixes for MPL and copywrite bot

### PR DESCRIPTION
**Pull request by modrake**
This implements a workaround for the copywrite bot's inability to maintain multiple licenses per repo. It creates subdirectory-specific configuration files in MPL-licensed subdirs and a new make target that applies copywrite to each impacted subdir after a repo-wide copywrite run. 

Copywrite GHA workflow is modified to use new make target, and old exclusions are commented out of the repo-wide copywrite.hcl configuration file as they would block execution of the MPL-adjusted fix. Lastly, this change includes a handful of files that were found missing headers on my initial execution of this change.

All included copywrite bot executions use the --plan flag on the original GHA workflow for Nomad.

### CODE REVIEW

**Review by tgross — COMMENTED** [review]
Thanks @modrake !

File: .copywrite.hcl [hunk 1]
```diff
@@ -9,9 +9,9 @@ project {
     "command/agent/bindata_assetfs.go",
     # licensed under MPL - ignoring for now until the copywrite tool can support
     # multiple licenses per repo.
-    "api/**",
-    "drivers/shared/**",
-    "plugins/**",
+    #"api/**",
+    #"drivers/shared/**",
+    #"plugins/**",
```
[the remark this answers is not in the document]
Comment (modrake) [reply .copywrite.hcl]: You bet - please give the updated change a quick look at your convenience to verify I correctly understood that your ask here was to simply make the ui/node_modules subdir a copywrite exception.

[the remark this answers is not in the document]
Comment (tgross) [reply .copywrite.hcl]: Yup, that works.

**Comment by tgross** [comment]
There's a failing check right now because the new HCL files weren't [`hclfmt`](https://github.com/fatih/hclfmt)'d, but once that's done this should be good to merge!

**Review by jrasell — APPROVED** [review]
LGTM, thanks @modrake!

**Pull request merged**